### PR TITLE
doc: typo/formating fixes

### PR DESCRIPTION
**Pull request by drunkard**
## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [x] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [ ] Updates relevant documentation
  - [x] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Review by sebastian-philipp — APPROVED** [review]

**Comment by sebastian-philipp** [comment]
jenkins test api

**Comment by drunkard** [comment]
I just pushed 2 commits today (2021-12-10), and 4 commits about two weeks ago, when I create PR I saw the previous PR, and can't create a new one, but previous 4 commits has been lost (``git push --force``) from my fork, and they are not in `master` branch yet, so, here's question:

1. have they (4 commits) been merged? (I don't known which branch merged them)
2. do I have to pick them back?

Sorry, I'm not very familiar with workflow on github.

**Comment by drunkard** [comment]
I reapplied those 4 commits, sorry for bothering you.

**Review by sebastian-philipp — APPROVED** [review]

**Comment by sebastian-philipp** [comment]
needs rebase

**Comment by drunkard** [comment]
Rebase done, conflict resolved.

**Comment by tchaikov** [comment]
rebased against main HEAD.

**Review by tchaikov — APPROVED** [review]

**Comment by tchaikov** [comment]
jenkins test make check

File: doc/rados/troubleshooting/troubleshooting-osd.rst [hunk 1]
```diff
@@ -174,7 +174,7 @@ If you start your cluster and an OSD won't start, check the following:
   bother by raising ``nf_conntrack_max`` to a much higher value via ``sysctl``.
   Be sure to raise ``nf_conntrack_buckets`` accordingly to
   ``nf_conntrack_max / 4``, which may require action outside of ``sysctl`` e.g.
-  ``"echo 131072 > /sys/module/nf_conntrack/parameters/hashsize``
+  ``echo 131072 > /sys/module/nf_conntrack/parameters/hashsize``.
```
Comment (anthonyeleven) [inline doc/rados/troubleshooting/troubleshooting-osd.rst:177]: Doh! This was probably my goof 😞

File: doc/radosgw/rgw-cache.rst [hunk 2]
```diff
@@ -1,155 +1,158 @@
...
 .. versionadded:: Octopus
 
 .. contents::
 
-This feature adds to RGW the ability to securely cache objects and offload the workload from the cluster, using Nginx.
-After an object is accessed the first time it will be stored in the Nginx cache directory.
-When data is already cached, it need not be fetched from RGW. A permission check will be made against RGW to ensure the requesting user has access.
-This feature is based on some Nginx modules, ngx_http_auth_request_module, https://github.com/kaltura/nginx-aws-auth-module, Openresty for Lua capabilities.
+This feature adds to RGW the ability to securely cache objects and offload
+the workload from the cluster, using Nginx.
```
Comment (anthonyeleven) [inline doc/radosgw/rgw-cache.rst:10]: I suggest changing "the workload" to just "workload", i.e. remove the "the".

File: doc/radosgw/rgw-cache.rst [hunk 3]
```diff
@@ -1,155 +1,158 @@
...
-Currently, this feature will cache only AWSv4 requests (only s3 requests), caching-in the output of the 1st GET request
-and caching-out on subsequent GET requests, passing thru transparently PUT,POST,HEAD,DELETE and COPY requests.
+After an object is accessed the first time it will be stored in the Nginx
+cache directory. When data is already cached, it need not be fetched from RGW.
+A permission check will be made against RGW to ensure the requesting user
+has access. This feature is based on some Nginx modules, ngx_http_auth_request_module,
+`ngx_http_aws_auth <https://github.com/kaltura/nginx-aws-auth-module>`_,
+Openresty for Lua capabilities.
 
+Currently, this feature will cache only AWSv4 requests (only s3 requests),
```
Comment (anthonyeleven) [inline doc/radosgw/rgw-cache.rst:19]: s/s3/S3/

File: doc/radosgw/rgw-cache.rst [hunk 4]
```diff
@@ -1,155 +1,158 @@
...
+your environment:
 
-`nginx-lua-file.lua` should go to `/etc/nginx/nginx-lua-file.lua`
+ * All Nginx conf files are under: https://github.com/ceph/ceph/tree/master/examples/rgw-cache
+ * `nginx.conf` should go to `/etc/nginx/nginx.conf`
+ * `nginx-lua-file.lua` should go to `/etc/nginx/nginx-lua-file.lua`
+ * `nginx-default.conf` should go to `/etc/nginx/conf.d/nginx-default.conf`
 
-`nginx-default.conf` should go to `/etc/nginx/conf.d/nginx-default.conf`
+The parameters that are most likely to require adjustment according to
```
Comment (anthonyeleven) [inline doc/radosgw/rgw-cache.rst:107]: Suggest s/according to/to match/

**Review by anthonyeleven — CHANGES_REQUESTED** [review]
By and large LGTM.

I did make several request for tweaks.

**Pull request closed without merging**